.github/workflows/main.yml

```
@@ -24,9 +24,8 @@ jobs:
     - name: Install Dependencies
       run: |
         apt update
-        apt install -y libgala-dev libgee-0.8-dev libglib2.0-dev libgranite-dev libgtk-3-dev libhandy-1-dev libcanberra-dev
         apt install -y libcloudproviders-dev libdbus-glib-1-dev libnotify-dev libpango1.0-dev libplank-dev
-        apt install -y libsqlite3-dev libunity-dev libzeitgeist-2.0-dev policykit-1 libgit2-glib-1.0-dev
+        apt install -y libgala-dev libgee-0.8-dev libglib2.0-dev libgranite-7-dev libgtk-4-dev libadwaita-1-dev libhandy-1-dev libcanberra-dev libsqlite3-dev
```

There's no Hdy in Gtk4 so we don't need that :)
```suggestion
        apt install -y libgala-dev libgee-0.8-dev libglib2.0-dev libgranite-7-dev libgtk-4-dev libadwaita-1-dev libcanberra-dev libsqlite3-dev
```